### ISSUE TITLE
Centerboxes' use of h3 tag violates accessibility "rules"

### ISSUE THREAD

**Issue by scottcwilson** [lat9/ZCA-Bootstrap-Template#328]
Files in `includes/modules/bootstrap/centerboxes` are using `h3` headings but there are no `h2` headings above them ... so they should be `h2`.  Not doing this is an accessibility error per Google Lighthouse.

Only happens for sites using centerboxes, of course.

**Comment by dbltoe** [comment]
You may find that sites in this situation have not looked at https://docs.zen-cart.com/user/new_user_topics/congratulations/ and the warning of what happens if they set the HEADING_TITLE_NESTED to ''

**Comment by scottcwilson** [comment]
Fair, but many folks don't want that on their front page.  It would probably be  easiest of all to change those two things to not use `<h2>` to prevent this.  Otherwise the centerboxes would. have to use `h2` or `h3` depending on the greeting. 

Here's a screenie of the Lighthouse warning: 
<img width="795" alt="accessibility" src="https://github.com/lat9/ZCA-Bootstrap-Template/assets/4391638/409dbd04-8b5d-4a37-8fc4-98ceccd742c5">

**Comment by lat9** [comment]
Probably the 'easiest' change while keeping the same look/feel is to use `<p class="h3"></p>` as a replacement.

**Comment by lat9** [comment]
> Then the same needs to be done to the categories heading as it’s an h3. Something I had hoped to address already but swamped. I am still inclined to produce compliance from the git go and place the onus of non-compliance on the customizer.
A pointer to where that "categories heading" resides?

**Comment by scottcwilson** [comment]
He's talking about the sideboxes in responsive classic; outside the scope of this issue.  

John, I'm not seeing this but if you can reliably dup it please open a new issue in the main Zen Cart project.

**Comment by dbltoe** [comment]
haredo.com is 1.5.8a bootstrap with a workround; ada4zencart is 2.0.0 bootstrap with no workaround.  Both SiteImprove and Wave show ADA4 starting with h4  Sigteimprove is the only one that really complains up front.

**Comment by scottcwilson** [comment]
John, your post is not related to the styling of Centerboxes.  **This issue is about styling of Centerboxes**.  
Please open a new issue if you want to talk about markup in another part of the page.
